workease.rb

```diff
@@ -102,6 +103,8 @@ def check_device(keyboard_id, mouse_id)
   def check(part)
     @semaphore.synchronize do
       time = Time.now.to_i
+      next if time < @pause_until
```

Comment:
Why was this added? Some sort of optimization?

Reply:
This stops the check method from running while a warning is displayed. So you can click close, without that click generating another warning.